### PR TITLE
Fix DBLookup and DBReport mediator scroll issues

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Fixing DBLookup and DBReport mediator scroll issues.

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java [hunk 1]
```diff
@@ -652,9 +653,10 @@ public void updatePart(Notification msg) {
 	 * 
 	 */
 	public String getHelpContent(Object key, int kind) {
-		if (key == EsbViewsRepository.DBLookupMediator.Statements.sqlStatements)
-			return "SQL Statements"; //$NON-NLS-1$
-		return super.getHelpContent(key, kind);
+//		if (key == EsbViewsRepository.DBLookupMediator.Statements.sqlStatements)
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java]: Remove commented lines.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java [hunk 2]
```diff
@@ -667,9 +668,10 @@ public void updatePart(Notification msg) {
 	 * 
 	 */
 	public String getHelpContent(Object key, int kind) {
-		if (key == EsbViewsRepository.DBReportMediator.Statements.sqlStatements)
-			return "SQL Statements"; //$NON-NLS-1$
-		return super.getHelpContent(key, kind);
+//		if (key == EsbViewsRepository.DBReportMediator.Statements.sqlStatements)
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java]: Remove commented lines.

Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java (hunk 1 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

**Pull request merged**